cloud_cleaner/resources/server.py

```diff
@@ -111,6 +127,73 @@ def __right_name(self, target):
                self.__name.match(target.name)
 
     def __right_age(self, target):
-        system_age = datetime.strptime(target.created, DATE_FORMAT)
+        system_age = datetime.strptime(target.launched_at, DATE_FORMAT)
         system_age = system_age.replace(tzinfo=utc)
         return self._now > (system_age + self._interval)
+
+    def __set_flagged(self):
+        """
+        Processes the txt file associated with the cloud to put all flagged
+        servers into the __flagged field.
+
+        :return: None
+        """
+        self.__flagged = []
+        conn = self._get_conn()
+        # List of server names that have been flagged to possibly be deleted
+        delete_list = []
+        # Open reader for the flagged file
+        reader = open(conn.name + "_flagged", 'r+')
+        for line in reader:
+            # delete the newline character at the end of the line
+            delete_list.append(line[:-1])
+
+        for id in delete_list:
+            server = conn.get_server(id)
+            self.__flagged.append(server)
+        reader.write("")
+        reader.close()
+
+    def write_to_flagged(self):
+        """
+        Writes the ids of flagged servers to the correct _flagged.txt file,
+        then sends emails if the flag is set in emails.json.
+        """
+        conn = self._get_conn()
+        writer = open(conn.name + "_flagged", 'w')
```

Comment:
Same comment as the previous method regarding the naming and using a context manager.